modules/ROOT/pages/index.adoc

```diff
@@ -6,34 +6,23 @@ endif::[]
-
-When your microservices can exist outside of the MuleSoft ecosystem, built using different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
-
-Most importantly, Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. You can discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
-
 == Anypoint Service Mesh Architecture
 
-A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job. This allows for providing security to your services. The manner in which these sidecar proxies interact is governed by an overall mixer, where a user can apply different policies and security.
+A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes Cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job.  
+
+The sidecar proxy enables you to bring your existing non-Mule microservices from the Kubernetes Cluster as APIs into Anypoint Platform. You can then manage these APIs within API Designer, API Manager, and Exchange. Additionally, you can monitor the health of your APIs using API analytics.
 
-The mixer is a highly modular and extensible component, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.
+A mixer, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems, governs the sidecar proxies. Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.
 
-The mixer is architected on a general-purpose plug-in model, which provides flexibility in dealing with different infrastructure backend applications. These plug-in applications, known as adapters, enable the mixer to interface with different infrastructure backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on.
+The mixer is architected on a general-purpose plug-in model, which provides flexibility in dealing with different infrastructure backend applications. These plug-in applications, known as adapters, enable the mixer to interface with the backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on.
```

Comment:
```suggestion
The mixer follows a general-purpose, plug-in model, which provides flexibility in working with different infrastructure backend applications. These plug-in applications, known as _adapters_, enable the mixer to interface with the backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on:
```
Does the mixer plug in to the applications, or vice versa? Consider a simpler rewording, like "the mixer plugs in to different backend applications, known as _adapters_, so that it can work with them to deliver core....